json/exp.json

```diff
@@ -9,8 +13,19 @@
         "verilator",
         "xgboost"
     ],
-    "experiment": "slurm_experiment",
-    "docker_home": "/<path>/<to>/slurm_home",
+    "_experiment": "experiment name",
+    "experiment": "exp",
+    "_simulation_mode": "1: execution-driven simulation w/o SimPoint, 2: trace-based simulation w/o SimPoint, 3: execution-driven simulation w/ SimPoint, 4: trace-based simulation w/ SimPoint, 5: PT-trace-based simulation w/ SimPoint",
+    "simulation_mode": "4",
+    "_workload_manager": "manual: run each exp scenario with all the simpoints at a time, slurm: Slurm schedules simulations based on its policy",
+    "workload_manager": "manual",
+    "_root_dir": "/<path>/<to>/<docker_home>",
```

Comment:
is this the root dir that is mounted in the docker container? If yes, rephrase:
"Root directory mounted into the docker container, for instance, the user's home directory."